packages/firestore/src/core/sync_engine.ts

```diff
@@ -42,7 +42,7 @@ import { Query } from './query';
 import { SnapshotVersion } from './snapshot_version';
 import { TargetIdGenerator } from './target_id_generator';
 import { Transaction } from './transaction';
-import { BatchId, ProtoByteString, TargetId } from './types';
+import { BatchId, ProtoByteString, TargetId, OnlineState } from './types';
```

Comment:
Did you `prettier` this? (I'm surprised OnlineState sorts after ProtoByteString.)

Reply:
Yes, I did, but it sucks at imports I guess. I fixed it manually, but going forward we should probably just not worry about it if it isn't automated. :-/

Reply:
Yuck.